cmd/migrate/executor/cstor_pool.go

```diff
@@ -77,18 +77,22 @@ func (m *MigrateOptions) RunCStorSPCMigrate() error {
 
 	klog.Infof("Migrating spc %s to cspc", m.spcName)
 	migrator := cstor.CSPCMigrator{}
+	cspcName := m.spcName
 	if m.cspcName != "" {
 		klog.Infof("using custom cspc name as %s", m.cspcName)
 		migrator.SetCSPCName(m.cspcName)
+		cspcName = m.cspcName
 	}
 	err := migrator.Migrate(m.spcName, m.openebsNamespace)
 	if err != nil {
 		klog.Error(err)
 		return errors.Errorf("Failed to migrate cStor SPC : %s", m.spcName)
 	}
 	klog.Infof("Successfully migrated spc %s to cspc", m.spcName)
-	klog.Infof("Make sure to migrate the associated PVCs, " +
-		"to find the old PVCs use `kubectl get cstorvolume.openebs.io -n <openebs-namespace>`, " +
-		"and to get the migrated PVCs use `kubectl get cstorvolume.cstor.openebs.io -n <openebs-namespace>`")
+
+	klog.Infof("Make sure to migrate the associated PVs, "+
+		"to list CStorVolume for the PVs which are pending migration use `kubectl get cstorvolume.openebs.io -n %s -l=openebs.io/storage-pool-claim=%s`, "+
+		"and to list CStorVolume for the migrated PVs use `kubectl get cstorvolume.cstor.openebs.io -n %s -l=openebs.io/cstor-pool-cluster=%s`",
```

Comment:
Are we keeping the CSPC name in CStorVolume labels?  If I remember correctly we are keeping on CVC [labels](https://github.com/openebs/cstor-csi/blob/182205e9591915146e57b5c502720075d974a39e/pkg/utils/maya.go#L121) but not on CV [labels](https://github.com/openebs/cstor-operators/blob/1c94fc00fc5b5ac7c1cc1ffa460219041fc5beb9/pkg/controllers/cstorvolumeconfig/volume_operations.go#L174).

I think we need to get PV name from CVC(`kubectl get cvc -l openebs.io/cstor-pool-cluster=<cspc-name> -o jsonpath='{.items[*].name}'`) (WDYT)?

Reply:
The PV names are only required for the non-csi volumes that are to be migrated. The migrated one is just to let the user know there are 2 cstorvolume apis.

Reply:
But the labels are not present on the new CV so the command will not work so the comment is valid.

Reply:
So we can just remove the -l from the command.